source/content/guides/new-dashboard/02-workspaces.md

```diff
@@ -62,6 +62,36 @@ Manage team members and assign [roles and permissions](/change-management#roles-
 
 Once it's enabled, use it to visually compare changes to the site. Visit the [Autopilot documentation](/guides/autopilot) for more information.
 
+### Webhooks
```

Comment:
I actually think this section fits better under "Settings" instead of Autopilot since it can be used across all sites, not just Autopilot-enabled sites. 

We should also include a new section within /guides/autopilot/ called "Notifications" and include Webhooks as well as the existing Email Notifications info (/guides/autopilot/enable-autopilot/#enable-autopilot-email-notifications).